src/log.js

```diff
@@ -82,7 +98,7 @@ class Log extends GSet {
     // Take the given key as the clock id is it's a Key instance,
     // otherwise if key was given, take whatever it is,
     // and if it was null, take the given id as the clock id
-    const clockId = (key && key.getPublic) ? key.getPublic('hex') : (key ? key : this._id)
+    const clockId = this._identity ? this._identity.id : this._id
```

Comment:
We should use `identity.publicKey` here as the clock ID (unique per device, otherwise entries created by the same ID on different devices might get a wonky order).